public/manifest.json

```diff
@@ -1,20 +1,20 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "DFX Exchange",
+  "name": "DFX MetaMask Exchange",
   "icons": [
     {
       "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
+      "sizes": "256x256",
       "type": "image/x-icon"
     },
     {
       "src": "logo.png",
       "type": "image/png",
-      "sizes": "72x72"
+      "sizes": "116x95"
```

Comment:
why those changes? does this fix something? if yes, what exactly?

Reply:
![image](https://user-images.githubusercontent.com/85513542/207728106-0fcac983-1cbd-4457-857e-bba275cad054.png)